README.md

```diff
@@ -1,17 +1,112 @@
 
-## Security
+## Introduction
+We construct a new large-scale benchmark termed *BigDetection*. Our goal is to simply leverage the training data from existing datasets ([LVIS](https://www.lvisdataset.org/), [OpenImages](https://storage.googleapis.com/openimages/web/index.html) and [Object365](https://www.objects365.org/overview.html)) with carefully designed principles, and curate a larger dataset for improved detector pre-training.
 
-See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
 
-## License
+![](./resources/bigdet_statistics.png)
 
-This project is licensed under the Apache-2.0 License.
+
+## Partial Results and Models
+
+### BigDetection Pretrained Models
+| Method | Lr Schd | mAP (bigdet val) | Model |Config |
+| --- | :---: | :---: | :---: | :---: |
+| YOLOv3 | 8x |  9.7 | - | [Config](configs/BigDetection/yolov3/yolov3_d53_mstrain-608_8x_bigdet.py) |
+| Deformable DETR | 8x | 13.1 | - | [Config](configs/BigDetection/deformable_detr/deformable_detr_r50_16x2_8x_bigdet.py) |
+| CBNetV2 | 3e | - | - | [Config](configs/BigDetection/cbnetv2/htc_cbv2_swin_base_giou_4conv1f_adamw_bigdet.py) |
+
+### CBNetV2 Finetuned on COCO
+| Method | mAP (coco minival/test-dev) | Model |Config |
+| --- |  :---: | :---: | :---: |
+| CBNetV2-Swin-B | 59.1/59.5 | - | [Config](configs/BigDetection/cbnetv2/htc_cbv2_swin_base_giou_4conv1f_adamw_20e_coco.py) |
+| CBNetV2-Swin-B (TTA) | 59.5/59.8 | - | [Config (test only)](configs/BigDetection/cbnetv2/htc_cbv2_swin_base_giou_4conv1f_adamw_20e_coco_tta.py) |
+
+### Notes
+- `TTA` denotes test time augmentation.
+- Pre-trained models of Swin Transformer can be downloaded from [Swin Transformer for ImageNet Classification](https://github.com/microsoft/Swin-Transformer).
+
+## Getting Started
+
+### Requirements
+- `Ubuntu 16.04`
+- `CUDA 10.2`
+
+### Installation
+```
+# Create conda environment
+conda create -n bigdet python=3.7 -y
+conda activate bigdet
+
+# Install Pytorch
+conda install pytorch==1.8.0 torchvision==0.9.0 cudatoolkit=10.2 -c pytorch
+
+# Install mmcv
+pip install mmcv-full==1.3.9 -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.8.0/index.html
+
+# Clone and install
+git clone https://github.com/cailk/BigDetection-MMDet
+cd BigDetection-MMDet
+pip install -r requirements/build.txt
+pip install -v -e .
+```
+
+### Data Preparation
+Our BigDetection involves 3 datasets and train/val data can be downloaded from their official website ([Objects365](https://www.objects365.org/download.html), [OpenImages v6](https://storage.googleapis.com/openimages/web/download.html), [LVIS v1.0](https://www.lvisdataset.org/dataset)). All datasets should be placed under $bigdetection/data/ as below. Contact us with [lkcai20@fudan.edu.cn](lkcai20@fudan.edu.cn) to get access to our pre-processed annotation files.
```

Comment:
You can put the class names here, which is just a simple .text files. The .json files will be provided as requested.